lib/utils/ember.js

```diff
@@ -223,6 +226,20 @@ function isEmberCoreModule(context, node, moduleName) {
   return false;
 }
 
+function isAnyEmberCoreModule(context, node) {
+  return (
+    isEmberComponent(context, node) ||
+    isGlimmerComponent(context, node) ||
+    isEmberController(context, node) ||
+    isEmberMixin(context, node) ||
+    isEmberRoute(context, node) ||
+    isEmberService(context, node) ||
+    isEmberArrayProxy(context, node) ||
+    isEmberObjectProxy(context, node) ||
```

Comment:
Should `EmberObject` be added to this list as well?

Reply:
I was thinking about that. I have now added it.